packages/yoroi-extension/app/api/jormungandr/lib/state-fetch/mockNetwork.js

```diff
@@ -575,7 +576,7 @@ export function toRemoteByronTx(
   const signedTx = RustModule.WalletV2.SignedTransaction
     .from_bytes(Buffer.from(request.signedTx, 'base64'));
   const hash = signedTx.id();
-  const transaction = signedTx.to_json().tx;
+  const transaction = JSONBigInt.parse(signedTx.to_json()).tx;
```

Comment:
Should not change parsing in this case. Here the `signedTx` is produced by `RustModule.WalletV2`, which is related to Cardano. Only values produced by `RustModule.SigmaRust` need updating.

Reply:
Oh, right. Didn't realize that.